Src/Particle/AMReX_ParticleContainer.H

```diff
@@ -576,6 +576,13 @@ public:
 
     Vector<Long> NumberOfParticlesInGrid  (int level, bool only_valid = true, bool only_local = false) const;
 
+    /**
+     * \brief Return capacity of memory for particles at specific grid
+     *  Level is defaulted to 0.
+     */
+
+    void CapacityOfParticlesInGrid  (LayoutData<std::size_t>& mem, int lev = 0) const;
```

Comment:
It's probably a good idea to make `int lev` required.
```suggestion
    void CapacityOfParticlesInGrid  (LayoutData<std::size_t>& mem, int lev) const;
```